src/copy.js

```diff
@@ -13,25 +13,33 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
 const FIX_TARGET = /twitter\.com|x\.com/g
 
 document.addEventListener('copy', e => {
-  target = window.getSelection().toString()
-
-  if (!target) {
-    return;
+  // ctrl+c or cmd+c copy
+  target = document.location.href;
```

Comment:
To avoid complexity, define each behavior separately window.getSelection().toString() and document.location.href